document/core/syntax/types.rst

```diff
@@ -367,6 +367,10 @@ Index Type
      \I32 ~|~ \I64 \\
    \end{array}
 
+.. _aux-idxtype-min:
+
+The minimum of two index types is defined as |I32| if either of the types are
+|I32| and |I64| otherwise.
```

Comment:
```suggestion
.. _aux-idxtype-min:

Convention
..........

The *minimum* of two index types is defined as |I32| if either of the types are |I32|,
and |I64| otherwise.

.. math::
   \begin{array}{llll}
   \itmin(\I64, \I64) &=& \I64 \\
   \itmin(\X{it}_1, \X{it}_2) &=& \I32 & (\otherwise) \\
   \end{array}
```
Plus define itmin as a macro pointing here.